architecture_docs/03_context_and_scope.md

```diff
@@ -0,0 +1,11 @@
+# Context and Scope
+
+This section puts Loculus into context with the outside world and defines the scope of the project.
+All external participants are listed in the diagram below:
+
+![Context View](plantuml/03_context_view.svg)
+
+We provide instructions how to install Loculus, and we host instances ourselves,
+but other maintainers can host their own instances as well.
+We offer guidance and documentation and are open for feature requests, but we do not provide direct support for custom instances.
+We cannot forsee all possible configurations and environments that Loculus might be deployed in.
```

Comment:
Does someone have a suggestion how to define the scope better? What's in the scope, what's not in the scope?

Reply:
That's a difficult question, I think that the scope isn't very clear yet and it will take more time and exchange with potential users to gain clarity.